### PR TITLE
Add IAC Scanning to left nav menu

### PR DESCRIPTION
**Pull request by drichards-87**
### What does this PR do? What is the motivation?

Removes IAC scanning from private beta and adds it to the left nav menu.

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: config/_default/menus/main.en.yaml [hunk 1]
```diff
@@ -5796,16 +5796,21 @@ menu:
       parent: csm_agentless
       identifier: csm_agentless_deployment_methods
       weight: 102
+    - name: IAC Scanning
```
Comment (cliffk) [inline config/_default/menus/main.en.yaml:5799]: @drichards-87 one nit here, let's lowercase the "a" in IaC, thanks.

**Comment by cliffk** [comment]
@drichards-87 also another thought, in the blog and inbound form we call it "IaC Security", in docs currently we're calling it "IaC Scanning". do you think it's okay or should we consolidate to one? we use "IaC Security" in marketing because it's more broad and it gives us more space to talk about adjacent things and not just scanning, but for docs it's specifically for scanning hence the name. im nitpicking here but just posing the question in the spirit of being consistent (since we have this problem all around security)

File: config/_default/menus/main.en.yaml (hunk 1 above)
Comment (drichards-87) [reply config/_default/menus/main.en.yaml:5799]: I'll update this in the PR.

**Comment by drichards-87** [comment]
> @drichards-87 also another thought, in the blog and inbound form we call it "IaC Security", in docs currently we're calling it "IaC Scanning". do you think it's okay or should we consolidate to one? we use "IaC Security" in marketing because it's more broad and it gives us more space to talk about adjacent things and not just scanning, but for docs it's specifically for scanning hence the name. im nitpicking here but just posing the question in the spirit of being consistent (since we have this problem all around security)

I can see both sides to this. However, I double-checked the UI and it looks like we refer to it as Infrastructure as Code (IaC) Scanning. So I'm in favor of keeping it as IaC Scanning. Thoughts?

Also, I will create a Jira card for moving the setup instructions to the main CSM setup docs. That's where we have the IaC Remediation setup instructions, so we should probably move the Scanning instructions there, as well.

**Comment by cliffk** [comment]
> > @drichards-87 also another thought, in the blog and inbound form we call it "IaC Security", in docs currently we're calling it "IaC Scanning". do you think it's okay or should we consolidate to one? we use "IaC Security" in marketing because it's more broad and it gives us more space to talk about adjacent things and not just scanning, but for docs it's specifically for scanning hence the name. im nitpicking here but just posing the question in the spirit of being consistent (since we have this problem all around security)
> 
> @cliffk I can see both sides to this. However, I double-checked the UI and it looks like we refer to it as Infrastructure as Code (IaC) Scanning. So I'm in favor of keeping it as IaC Scanning. Thoughts?
> 
> Also, I will create a Jira card for moving the setup instructions to the main CSM setup docs. That's where we have the IaC Remediation setup instructions, so we should probably move the Scanning instructions there, as well.

Cool, let's leave it iac scanning. This PR is good to go! Thanks

**Review by cliffk — APPROVED** [review]

File: content/en/security/cloud_security_management/iac_scanning.md [hunk 2]
```diff
@@ -1,13 +1,18 @@
 ---
 title: IaC Scanning
-private: true
 ---
 
-<div class="alert alert-info">Static Infrastructure as Code (IaC) scanning is in Preview.</div>
+{{< callout url="https://www.datadoghq.com/product-preview/iac-security/" >}}
```
Comment (estherk15) [inline content/en/security/cloud_security_management/iac_scanning.md:5]: Just verifying before you merge, this link didn't work for me.

**Review by estherk15 — APPROVED** [review]

**Pull request merged**